src/product.html

```diff
@@ -62,18 +62,18 @@
           </section>
         </header> <!-- Risky Business Ends here -->
         <section class="col col-2-3"> <!-- Left Column Starts Here -->
-          <section class="img-slider" ng-app="slider-module"> <!-- Air Drummers Starts here -->
+          <section class="img-slider"> <!-- Air Drummers Starts here -->
             <div class="img-main">
               <a href="#" id="prev"><div class = "arrow-case"></div><i class="fa fa-arrow-left"></i></a>
-              <img class= "img-active" src={{listingImages.results[0].url_570xN}}>
+              <img class= "img-active" ng-src={{listingImages.results[0].url_570xN}}>
```

Comment:
I knew that, I was trying to hit the goal of at least getting the API and making it appear to the our page. using `MainImage` is definitely a proper choice. I wish I have time, haha.